javatests/arcs/core/entity/HandleManagerTestBase.kt

```diff
@@ -739,24 +761,36 @@ open class HandleManagerTestBase {
     @Test
     open fun collection_dereferenceEntity_nestedReference() = testRunner {
         // Create a stylish new hat, and create a reference to it.
-        val hatCollection = writeHandleManager.createHandle(
-            HandleSpec(
-                "hatCollection",
-                HandleMode.ReadWrite,
-                toType(
-                    Hat,
-                    HandleDataType.Entity,
-                    HandleContainerType.Collection
-                ),
-                setOf(Hat)
+        val hatSpec = HandleSpec(
+            "hatCollection",
+            HandleMode.ReadWrite,
+            toType(
+                Hat,
+                HandleDataType.Entity,
+                HandleContainerType.Collection
```

Comment:
Can we just put the correct type directly in? It's actually shorter to do so... `CollectionType(EntityType(Hat.SCHEMA))`

Reply:
Done